### PR TITLE
Fix the network.concat_ports() issue.

### PR DESCRIPTION
**Pull request by Asachoo**
Fix issue https://github.com/scikit-rf/scikit-rf/issues/1119, and add relevant unit test cases.

### CODE REVIEW

File: skrf/tests/test_network.py [hunk 1]
```diff
@@ -441,6 +441,14 @@ def test_cascade2(self):
         self.assertEqual(self.ntwk1 >> self.ntwk2, self.ntwk3)
         self.assertEqual(self.Fix2 >> self.DUT2 >> self.Fix2.flipped(), self.Meas2)
 
+    def test_concat_ports(self):
+        l2 = self.cpw.line(0.07, 'm', z0=50)
```
Comment (Ttl) [inline skrf/tests/test_network.py:445]: Store `l2` in class (`self`) rather than duplicating it here.

**Review by Ttl — APPROVED** [review]

File: skrf/tests/test_network.py (hunk 1 above)
Comment (Asachoo) [reply skrf/tests/test_network.py]: Changes made as requested.

File: skrf/network.py [hunk 2]
```diff
@@ -5482,6 +5482,7 @@ def f(x, y):
     ntwkC = ntwkA.copy()
     ntwkC.s = C
     ntwkC.z0 = np.hstack([ntwkA.z0, ntwkB.z0])
+    ntwkC.port_modes = np.hstack([ntwkA.port_modes, ntwkB.port_modes])
```
Comment (jhillairet) [inline skrf/network.py:5485]: Can you also fix the missing spaces in the docstring ? I believe you should only change

``` 
>>>concat([ntwkA,ntwkB])
>>>concat([ntwkA,ntwkB,ntwkC,ntwkD], port_order='second')
To put for lines in parallel
>>> from skrf import air
>>> l1 = air.line(100, z0=[0,1])
>>> l2 = air.line(300, z0=[2,3])
>>> l3 = air.line(400, z0=[4,5])
>>> l4 = air.line(400, z0=[6,7])
>>> concat_ports([l1,l2,l3,l4], port_order='second')
```

to

```
>>> concat([ntwkA,ntwkB])
>>> concat([ntwkA,ntwkB,ntwkC,ntwkD], port_order='second')

To put for lines in parallel

>>> from skrf import air
>>> l1 = air.line(100, z0=[0,1])
>>> l2 = air.line(300, z0=[2,3])
>>> l3 = air.line(400, z0=[4,5])
>>> l4 = air.line(400, z0=[6,7])
>>> concat_ports([l1,l2,l3,l4], port_order='second')
```

**Review by jhillairet — APPROVED** [review]

File: skrf/network.py (hunk 2 above)
Comment (Asachoo) [reply skrf/network.py:5486]: Changes made as requested.

**Pull request merged**